docs/hive-integration/kube-api-conditions.md

```diff
@@ -96,12 +96,12 @@ The Agent condition types supported are: `SpecSynced`, `Connected`, `ReadyForIns
 |Validated|False|ValidationsUserPending|The agent's validations are pending for user: "summary of not-succeeded validations"|If the host status is "pending-for-input"|
 |Validated|Unknown|ValidationsUnknown|The agent's validations have not yet been calculated|If the validations have not yet been calculated|
 ||||||
-|ReadyForInstallation|True|AgentIsReady|The agent is ready to begin the installation|If the host is approved and in status "known"|
-|ReadyForInstallation|False|AgentNotReady|The agent is not ready to begin the installation|If the host is before installation ("discovering"/"insufficient"/"disconnected"/"pending-input")|
-|ReadyForInstallation|False|AgentAlreadyInstalling|The agent cannot begin the installation because it has already started|If the agent has begun installing ("preparing-successful","preparing-for-installation", "installing") |
-|ReadyForInstallation|False|AgentIsNotApproved|The agent is not approved|If the host is not approved and in status "known"|
-|ReadyForInstallation|False|AgentAlreadyInstalling|The agent cannot begin the installation because it has already started|If the agent has begun installing ("preparing-successful","preparing-for-installation", "installing") |
-|ReadyForInstallation|False|AgentInstallationStopped|The agent installation stopped|If the agent has stopped installing ("installed", "error") |
+|RequirementsMet|True|AgentIsReady|The agent is ready to begin the installation|If the host is approved and in status "known"|
+|RequirementsMet|False|AgentNotReady|The agent is not ready to begin the installation|If the host is before installation ("discovering"/"insufficient"/"disconnected"/"pending-input")|
+|RequirementsMet|False|AgentAlreadyInstalling|The agent cannot begin the installation because it has already started|If the agent has begun installing ("preparing-successful","preparing-for-installation", "installing") |
```

Comment:
I think we need to rephrase the message.
In ACI we have: `The cluster requirements are met`

Reply:
@rollandf Do you mean to change the message `The agent cannot begin the installation because it has already started` to `The agent requirements are met`?